packages/components/src/components/form-submit-error-message/form-submit-error-message.tsx

```diff
@@ -1,20 +1,40 @@
-import React from 'react';
 import classNames from 'classnames';
 import Icon from '../icon/icon';
 import Text from '../text';
+import { Localize } from '@deriv/translations';
+import OpenLiveChatLink from '../open-livechat-link';
 
 type TFormSubmitErrorMessage = {
     className?: string;
     message: string;
+    error_code?: string;
 };
 
-const FormSubmitErrorMessage = ({ className, message }: TFormSubmitErrorMessage) => (
-    <div className={classNames('dc-form-submit-error-message', className)}>
-        <Icon icon='IcAlertDanger' data_testid='form_submit_error' />
-        <Text as='p' size='xxs' weight='bold' color='prominent'>
-            {message}
-        </Text>
-    </div>
-);
+const FormSubmitErrorMessage = ({ className, message, error_code }: TFormSubmitErrorMessage) => {
+    const getErrorMessage = () => {
+        switch (error_code) {
```

Comment:
switch case is only handling a single specific error code. i think simple if else case would be fine